### ISSUE TITLE
Bismuth high in blood

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10393]
**Preferred term label:**
Bismuth high in blood

**Synonyms**

**Definition (free text, please give PubMed ID)**
greater than 5µg/dL

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Bismuth toxicity

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000887
- label: Elevated blood bismuth concentration
- parents: [Blood xenobiotic (HP:0410172)](https://hpo.jax.org/app/browse/term/HP:0410172)
- pmid: 
- def: The concentration of bismuth in the blood is above the upper limit of a guidance value for total blood bismuth concentrations that implies possible injurious effects.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10393
- synonyms:
